fix some code explanations and minor issues (#668)
mlr-org/mlr3book: book/chapters/chapter13/beyond_regression_and_classification.qmd

autoplot(prediction, type = "threshold", measure = msr_costs)
```

> As expected, the optimal threshold is greater than 0.5 which leads the model to be more conservative in giving out credits.

Reviewer: ```suggestion
As expected, the optimal threshold is greater than 0.5 which leads the model to predicting 'bad' credit more often.
```
